modules/openvino-langchain/README.md

```diff
@@ -21,11 +21,17 @@ npm install openvino-langchain
 
 In order to use OpenVINO, you need to convert and compress the text generation model into the [OpenVINO IR format](https://docs.openvino.ai/2025/documentation/openvino-ir-format.html).
 
-Tested compatibility with:
-  - BAAI/bge-small-en-v1.5 (Embeddings model)
-  - openlm-research/open_llama_7b_v2
-  - meta-llama/Llama-2-13b-chat-hf
-  - microsoft/Phi-3.5-mini-instruct
+The following models are tested:
```

Comment:
In the  _openvino-langchain/sample/README.md_ there is a mention that `Phi-3.5-mini-instruct` includes embedding model. It also should be updated.

Reply:
Thanks, this row is removed!